Assets/MixedRealityToolkit/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -107,6 +110,14 @@ private void OnGUI()
             else
             {
                 RenderConfiguredConfirmation();
+
+                EditorGUILayout.Space();
+
+                showConfigurations = EditorGUILayout.Foldout(showConfigurations, "Configurations", true);
```

Comment:
The newly added code here and the code above (lines 104-107) are basically the same thing with the only difference being "Modify Configurations" and "Configurations"

Can we just move that stuff out of the if/else, so that it's common between the two, and just use the same text (i.e. modify configurations or configurations?)

Reply:
sure!